src/AspNet.Security.OAuth.EVEOnline/EVEOnlineAuthenticationServer.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace AspNet.Security.OAuth
+{
+    /// <summary>
+    /// Defines a list of available authentication endpoints based on the 
+    /// accessed server.
+    /// </summary>
+    public enum EVEOnlineAuthenticationServer {
+        Tranquility = 0,
```

Comment:
Consider adding some XML documentation to indicate the differences between these 2 values.